source/clear-linux/reference/reference.rst

```diff
@@ -3,11 +3,11 @@
 Reference
 #########
 
-This section provides additional information on the |CL| project and |CL|
-features.
+This section provides additional information for use of |CL-ATTR| and 
+its features.
```

Comment:
Optional:  Where `features` appears, you could add a link to our https://clearlinux.org/features page.